src/main/charts/crowd/templates/configmap-jmx-config.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.monitoring.exposeJmxMetrics }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}-jmx-config
+  labels:
+    {{- include "common.labels.commonLabels" . | nindent 4 }}
+data:
+{{ if .Values.monitoring.jmxExporterCustomConfig }}
```

Comment:
Wasn't possible to include this snippet in the `common` package?

Reply:
Indeed. Can be fixed in further PRs.

Reply:
Fixed